src/solvers/dg/3d/dg.jl

```diff
@@ -1307,11 +1319,120 @@ function rhs!(dg::Dg3D, t_stage)
   @timeit timer() "source terms" calc_sources!(dg, dg.source_terms, t_stage)
 end
 
+# Calculate volume integral and update u_t
+apply_positivity_preserving_limiter!(dg::AbstractDg) = apply_positivity_preserving_limiter!(dg.elements.u, dg, equations(dg))
+
+# Only implemented and tested for compressible Euler equations in 3D
+function apply_positivity_preserving_limiter!(u, dg::AbstractDg, equation::AbstractEquation)
+end
```

Comment:
```suggestion
```
I favor not implementing this. If a user chooses to apply the limiter for something else, it will silently not work. When we remove this definition, a `MethodError` will be thrown instead, making clear that the functionality isn't implemented yet.

Reply:
Then, I would check whether the limiter shall be applied before calling this function.

Reply:
I second that!